registry/quilt_server/views.py

```diff
@@ -1617,6 +1618,53 @@ def list_users():
 
     return resp.json()
 
+@app.route('/api/users/list_detailed', methods=['GET'])
+@api(enabled=ENABLE_USER_ENDPOINTS, require_admin=True)
+@as_json
+def list_users_detailed():
+    package_counts_query = (
+        db.session.query(Package.owner, sa.func.count(Package.owner))
+        .group_by(Package.owner)
+        )
+    package_counts = dict(package_counts_query) 
+
+    events = (
+        db.session.query(Event.user, Event.type, sa.func.count(Event.type))
+        .filter(Event.type.in_([Event.Type.INSTALL, Event.Type.PREVIEW]))
```

Comment:
I'd remove it and just return all of the data - there's such a small cost to it.